app/src/main/java/de/danoeh/antennapod/fragment/FeedInfoFragment.java

```diff
@@ -169,7 +186,7 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe(result -> {
                     feed = result;
-                    showFeed();
+                    loadStatistics();
```

Comment:
I think it would be better to call both here and then decouple displaying statistics and displaying the other feed info